### ISSUE TITLE
Custom RPC for EVMs

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-suite#9299]
We are currently investigating an option to support BSC and Polygon through Blockbook and external provider. That means we will run Blockbook for BSC and Polygon but we will connect these instances to Tatum / QuickNode / whatever (#8443).

This approach has some pros and cons:
:+1: Suite is ready
:+1: Performance
😐 Painful to maintain the nodes but outsourcing solves this
😐 We need new servers for BSC / Polygon (only one-time costs and some minor maintenance, so not a biggie)
:-1: Doesn't scale well, we need blockbook for each coin

This seems fine, but an interesting point came up: connecting to the nodes from Suite directly, would significantly simplify adding new EVMs. We could even support "a custom RPC" (as MetaMask) easily.

Connecting to nodes directly from Suite:
:+1: Scales well. Basically we will suddenly support all EVMs or with very little effort
:-1: Needs development and maintainability in Suite

Note, that I am by no means advocating for the latter. I just would like to have this researched so we know the benefits and drawbacks.

-----

**Tl;dr:** this issue is about investigating what would it entail to support EVMs directly from Suite. I guess another provider in blockchain-link would be needed.
- How much code would need to be written and maintained? Let's estimate in MDs.
- How long would the discovery take? Would it be usable?
- How many requests would it be? We can estimate the same visitors as for ETH for simplicity.

**Comment by mroz22** [comment]
probably solved for now, handing over to the trends team

**Comment by EuryShadow** [comment]
Custom RPCs are implemented in the desktop app; further follow-up issues are handled here: https://github.com/trezor/trezor-suite/issues/24047
